app/controllers/candidate_interface/additional_referees_controller.rb

```diff
@@ -125,6 +130,14 @@ def redirect_to_dashboard_if_no_references_needed
       redirect_to candidate_interface_application_form_path
     end
 
+    def redirect_to_contact_support_if_at_maximum_reference_count
+      redirect_to candidate_interface_additional_referee_contact_support_path if at_maximum_referee_count && reference_status.needs_to_draft_another_reference?
+    end
```

Comment:
Why is `reference_status.needs_to_draft_another_reference?` necessary?

Reply:
it isn't you're right 👍